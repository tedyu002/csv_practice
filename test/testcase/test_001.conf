CSV_INPUT=test_001.conf.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=H1 CHAR(4),H2 VARCHAR(32),H3 VARCHAR(12),H4 INTEGER,H5 BOOL,H6 DATETIME,H7 DOUBLE
CSV_FORMULA=Print H1[1] + H2[2] + H3[3]
SORT_HEADERS=H4,H2,H6,H1
SORT_ORDER=ASC
