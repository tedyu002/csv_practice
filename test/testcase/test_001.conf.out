800000,"Ford",2017/06/03 12:30:01,"1997","E350",TRUE,12.35
858000,"Skoda",2017/05/15 07:48:37,"2016","Citigo",FALSE,172.80
950000,"Skoda",2017/04/13 02:18:52,"2017","Yeti",FALSE,12.52
958000,"Skoda",2017/05/15 07:47:34,"2017","Rapid",TRUE,15.63
1150000,"Mazda",2017/05/15 02:18:52,"2003","CX-5",FALSE,45.96
1550000,"BMW",2017/06/19 13:49:12,"2017","X1",TRUE,27.85
1550000,"Volvo",2017/05/15 09:08:12,"2016","V40",FALSE,69.63
1650000,"BMW",2017/06/10 19:41:30,"2015","X2",TRUE,26.36
1750000,"Volvo",2017/06/09 03:49:00,"2016","S60",FALSE,52.63
1800000,"Mercedes-Benz",1998/10/03 08:33:23,"2003","C220",TRUE,528.69
1850000,"Audi",2017/05/15 02:18:37,"1991","A8",TRUE,12.50
1850000,"BMW",2017/06/09 14:16:14,"2017","X3",TRUE,36.96
2050000,"BMW",2007/06/09 14:24:35,"2015","X4",TRUE,63.52
2250000,"Audi",2017/03/15 07:48:37,"1991","Q7",TRUE,15.69
2250000,"BMW",2017/06/09 14:26:52,"2017","320i",FALSE,63.96
2350000,"Volvo",2017/01/10 10:53:57,"2016","XC60",FALSE,12.63
2350000,"Volvo",2017/06/10 10:53:44,"2015","S80",FALSE,35.36
2450000,"BMW",2017/06/09 14:24:44,"2017","X5",FALSE,13.52
2550000,"BMW",2010/06/09 14:26:06,"2017","X6",TRUE,23.63
2550000,"Volvo",2017/05/12 08:52:28,"2016","XC90",TRUE,28.36
2550000,"Volvo",2017/06/10 12:41:15,"2016","XC70",TRUE,29.52
2800000,"Mercedes-Benz",2013/06/04 23:30:01,"1991","S320",TRUE,12.66
4350000,"Ferrari",2013/06/09 14:27:04,"2015","812",TRUE,125.63
